README.md

```diff
@@ -18,7 +18,7 @@ This repository provides a CMake-based build project for SLEIGH so that it can b
 
 | Name | Version | Linux Package to Install | macOS Homebrew Package to Install |
 | ---- | ------- | ------------------------ | --------------------------------- |
-| [Git](https://git-scm.com/) | Latest | git | N/A |
+| [Git](https://git-scm.com/) | Latest | git | git |
```

Comment:
The reason I made this N/A is that it's available via XCode Command Line Tools. That's the `git` that most people are going to be using on macOS.